WordPress/gutenberg
# Update Image Block's image classes with dimensions

## Description
Adds a class to the figure generated by the image block which corresponds to the slug of the image size selected in the inspector.

<img width="1146" alt="Screenshot 2019-05-07 at 18 37 30" src="https://user-images.githubusercontent.com/107534/57312981-506c0400-70f7-11e9-9044-06ec92692b5d.png">


## How has this been tested?
- insert an image block
- add an image
- from the inspector pick another size than the default
- save the post
- check the front end post result and inpect the image: the container figure element should have a class named `size-{yourchosensize}`.

## Types of changes
- updated the select control to allow for data attributes to be sent to it
- updated the image block to save both in the editor and in the content the image size class
